jekyll/_cci2/building-xcode-projects.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: classic-docs
+title: Using a macOS Build Image
+categories: [containerization]
+description: macOS
+changefreq: "weekly"
+---
+
+This document describes how to specify an Xcode project and use a macOS or OSX image for your build environment in the following sections:
+
+* TOC
+{:toc}
+
+## Overview of macOS Images
+
+CircleCI maintains a manifest of the software installed on the available OSX and macOS build images. This manifest includes the version of the software components including the operating system, Xcode, Python, and Ruby, for example.
+
```

Comment:
2.0 is available with 9.0 Beta and 8.3.3 images only, plus future images. We will still provide older images on 1.0. We need to update this to reflect that.